src/lang/proc_macros/client/mod.rs

```diff
@@ -162,6 +171,17 @@ impl ProcMacroClient {
     }
 }
 
+impl Debug for ProcMacroClient {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        f.debug_struct("ProcMacroClient")
+            .field("connection", &self.connection)
+            .field("id_generator", &self.id_generator)
+            .field("requests_params", &self.requests_params)
+            .field("error_channel", &self.error_channel)
+            .finish_non_exhaustive()
+    }
+}
+
```

Comment:
hmm, what about adding a dummy Debug impl to ProcMacroRequestTracker that prints just `ProcMacroRequestTracker { .. }`, wouldn't it make things simpler?

Reply:
Why not debug this as well though?

Reply:
I recall we discussed we do not want to debug its internals anyway. But anyhow, this is subjective idea, I'm not even attached to it myself, ignore this comment if you wish.